### PR TITLE
Teleporter changes [NON-MODUALR]

### PR DESCRIPTION
**Pull request by Gandalf2k15**
## About The Pull Request

Let's not go easy on dumb people that use an uncalibrated teleporter;
You'll lose 2 limbs, and you'll also receive a a lot of radiation.

## Why It's Good For The Game

Muh character.

## Changelog
:cl:
balance: Teleproters no longer change species, but are FAR more deadly when used uncalibrated.
/:cl:

### CODE REVIEW

**Comment by tf-4** [comment]
Ehhhh I think three seems like a lot. It was never an issue before, I'm not sure why it would become one now.

**Comment by Azarak** [comment]
I think 2 is great, 3 may be a bit much. With 3 they could be close to death due to bleeding fast? And 2 would give them enough time to suffer. Plus they'll be suffering from radiation poisoning

**Comment by ShadeAware** [comment]
> 
> 
> I think 2 is great, 3 may be a bit much. With 3 they could be close to death due to bleeding fast? And 2 would give them enough time to suffer. Plus they'll be suffering from radiation poisoning

As far as I know, TG does not have bleeding stump wounds, so losing a limb is essentially losing all of the damage connected to that limb and nothing else. I could be wrong though.

**Comment by Gandalf2k15** [comment]
Changed it to two limbs, and upped the rad damage.

**Comment by Azarak** [comment]
> 
> 
> > I think 2 is great, 3 may be a bit much. With 3 they could be close to death due to bleeding fast? And 2 would give them enough time to suffer. Plus they'll be suffering from radiation poisoning
> 
> As far as I know, TG does not have bleeding stump wounds, so losing a limb is essentially losing all of the damage connected to that limb and nothing else. I could be wrong though.

this will need to be fixed in another PR 👀

File: code/game/machinery/teleporter.dm [hunk 1]
```diff
@@ -77,13 +77,25 @@
...
+					var/dismembered = 0
+					to_chat(human, "<span class='danger'>Your limbs lose molecular cohesion as you teleport!</span>")
+					for(var/obj/item/bodypart/BP in human.bodyparts)
+						if(dismembered <= 1) //Remove 3 body parts.
+							if(BP.name == "chest" || BP.name == "head")
+								continue
+							BP.dismember()
+							qdel(BP) //WHEEEEE LOST TO SPACE N' TIME
+							dismembered++
+					//SKYRAT EDIT CHANGE END
```
Comment (Azarak) [inline code/game/machinery/teleporter.dm]:
```suggestion
						if(dismembered < 2) //Remove 2 body parts.
							if(BP.body_zone == BODY_ZONE_CHEST || BP.body_zone== BODY_ZONE_HEAD)
								continue
							BP.dismember()
							qdel(BP) //WHEEEEE LOST TO SPACE N' TIME
							dismembered++
						else
							break
					//SKYRAT EDIT CHANGE END
```

**Comment by Owai-Seek** [comment]
Doesn't this mean that the teleporter can just, behead you?

*Edit*

Nevermind, had to read more thoroughly.

**Pull request merged**